README.md

```diff
@@ -63,6 +63,7 @@ if your compiler does not support gcc-style options, if you are cross compiling,
 
 To enable CUDA support, add `CUDA_DIR=/opt/cuda` or an appropriate directory to your `make` invocation.
 To enable HIP support, add `ROCM_DIR=/opt/rocm` or an appropriate directory.
+To enable SYCL support, add `SYCL_DIR=/opt/sycl` or an appropriate directory.
```

Comment:
Don't we need to explicitly set compilers? This links for me on noether. In the other backends, we work out any necessary includes so that `CC` and `CXX` don't need to be special, but it's needed here. That's okay for this PR, but we should state that oneAPI compilers should be used for everything when building a SYCL backend.
```console
$ . /opt/intel/oneapi/setvars.sh
$ make SYCL_DIR=/opt/intel/oneapi/compiler/latest/linux SYCLCXX=icpx CC=icx CXX=icpx
```

Reply:
I haven't been explicitly setting compilers on Sunspot builds, but that's probably because `CC` and `CXX` are set correctly already.